### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ghost**
Just edited grammar and added a few things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h1 align="center">odysseyn1x</h1>
-<h3 align="center">Linux-based distro that lets you install checkra1n, odysseyra1n and Project Sandcastle.</h3>
+<h3 align="center">Linux-based distro that lets you install Checkra1n, Odysseyra1n and Project Sandcastle.</h3>
```
Comment (raspberryenvoie) [inline README.md]: I think checkra1n and odysseyra1n don't start with a capital letter. Check out these links: https://www.reddit.com/r/jailbreak/wiki/checkra1n#wiki_switch_from_checkra1n_to_odysseyra1n https://checkra.in/

File: README.md [hunk 2]
```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
...
 ### Installing Android
 Select Project Sandcastle > Setup Project Sandcastle > Start Android.
 When Project Sandcastle has been installed, simply choose "Start Android" to boot back into Android.
 
 **Removing Android:**
-If you wish to remove the Android NAND image and reclaim the space you can login via SSH to your checkra1ned device and mount the final volume and remove the nand file. To do this run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path you can then run as root (type `su`):
+If you wish to remove the Android NAND image and reclaim the space, you can login via SSH to your Checkra1ned device, and mount the final volume, and remove the NAND file. To do this, run `ls /dev/disk0s1s*` and find the last volume. You can verify it's the right volume by running `/System/Library/Filesystems/apfs.fs/apfs.util -p VOLUME_HERE` and if it says Android, that's the correct one. Once you have the volume path, you can then run these commands as root (type `su`):
```
Comment (raspberryenvoie) [inline README.md]: If I'm not wrong, checkra1ned is used as an adjective so it doesn't require a capital letter.

File: README.md [hunk 3]
```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
...
 git clone https://github.com/raspberryenvoie/odysseyn1x.git
 cd odysseyn1x
 sudo ./build.sh
 ```
 ## Credits
-- Asineth for checkn1x
-- The Checkra1n team for [checkra1n](https://checkra.in)
-- CoolStar for [odysseyra1n](https://github.com/coolstar/Odyssey-bootstrap)
-- [The Procursus team](https://github.com/ProcursusTeam/) for [Procursus](https://github.com/ProcursusTeam/Procursus)
+- Asineth for Checkn1x
```
Comment (raspberryenvoie) [inline README.md]: Checkn1x doesn't start with a capital letter: https://git.asineth.gq/asineth/checkn1x

File: README.md [hunk 4]
```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
...
+- The Checkra1n team for [Checkra1n](https://checkra.in)
+- CoolStar for [Odysseyra1n](https://github.com/coolstar/Odyssey-bootstrap)
+- [The Procursus Team](https://github.com/ProcursusTeam/) for [Procursus](https://github.com/ProcursusTeam/Procursus)
 - [Corellium](https://github.com/corellium) for [Project Sandcastle](https://projectsandcastle.org)
-- [u/GOGO307](https://www.reddit.com/user/GOGO307/) for the concept of an iso with odysseyra1n
-- [MyCatCondo](https://github.com/MyCatCondo) for suggesting to integrate Project Sandcastle (Android and Linux for the iPhone)
+- [u/GOGO307](https://www.reddit.com/user/GOGO307/) for the concept of an ISO with Odysseyra1n
+- [MyCatCondo](https://github.com/MyCatCondo) for suggesting to integrate Project Sandcastle (Android and Linux for the iPhone and maybe the iPad)
```
Comment (raspberryenvoie) [inline README.md]: Another thing, I think that Project Sandcastle doesn't support the iPad: https://projectsandcastle.org/status

File: README.md [hunk 5]
```diff
@@ -11,16 +11,16 @@ The `amd64` iso is for 64-bit CPUs (AMD and Intel) and the `i686` one is for 32-
 If you are unsure which one to download, the `amd64` iso will work in most cases.
 
 1. Download an `.iso` [here](https://github.com/raspberryenvoie/odysseyn1x/releases).
-2. Download [balenaEtcher](https://www.balena.io/etcher/).
+2. Download [balenaEtcher](https://www.balena.io/etcher/). As far as I know, Rufus will ***not*** work for reasons unexplained, possibly because Rufus may make too many partitions.
```
Comment (raspberryenvoie) [inline README.md]: Rufus works for some people, say it might not work instead.

File: README.md [hunk 6]
```diff
@@ -34,39 +34,39 @@ OpenSSH is installed by default, so please change the mobile/root passwords to p
 4. After that, run `passwd` to change its password.
 
 ## Project Sandcastle
-**What's Project Sandcastle:** It's Android and Linux for the iPhone. Projectsandcastle is in beta so not everything is fully supported. [More information](https://projectsandcastle.org)
+**What's Project Sandcastle:** It's Android and Linux for the iPhone. Project Sandcastle is in beta so not everything is fully supported. [More information](https://projectsandcastle.org) and (https://ipadlinux.org)
```
Comment (raspberryenvoie) [inline README.md]: What's ipadlinux?

**Review by raspberryenvoie — COMMENTED** [review]
Thanks you so much for your contribution!

**Comment by ghost** [comment]
Here, I’ll fix those problems xD and I’ll answer all of those questions. iPadLinux is a website where it talks about how to get Linux on an iPad. It’s not official, but it says there is shared hardware in an iPad that allows it to run Sandcastle. I honestly don’t know about Android Sandcastle, but Linux Sandcastle looks as if it would boot up fine, but there’s an error.
I’ll fix those grammar problems in a min lol, I’m just OCD and I didn’t know since I’m usually only on Reddit to check out r/jailbreak.
I did not know that Rufus sometimes works. I just used Rufus 11 (I’m thinking it’s 11, but nevertheless it’s the latest version) and  trying to boot it up just left me with a blinking cursor thing in a black screen that is completely non interactive. Etcher looks fine, so pardon me. xD
Those grammar errors will be fixed right away!

**Comment by raspberryenvoie** [comment]
Thanks a lot for all your contributions MyCatCondo! :smiley:

**Comment by ghost** [comment]
Np!

**Comment by raspberryenvoie** [comment]
I've just changed two small things:
- `Linux-based distro that lets you install checkra1n, odysseyra1n and Project Sandcastle.` (Remove the comma before and)
- `u/GOGO307 for the concept of an ISO with odysseyra1n` (Remove the capital letter of odysseyra1n)

**Comment by ghost** [comment]
Oh I missed those! Thanks!

**Comment by raspberryenvoie** [comment]
Np! Thanks *you*

**Pull request merged**